xarray/tests/test_plot.py

```diff
@@ -470,6 +471,16 @@ def test_center(self):
         assert cmap_params['levels'] is None
         assert cmap_params['norm'] is None
 
+    def test_cmap_option(self):
+        with xr.set_options(cmap_sequential='magma'):
+            cmap_params = _determine_cmap_params(self.data)
+            assert cmap_params['cmap'] == 'magma'
+
+    def test_cmap_divergent_option(self):
+        with xr.set_options(cmap_divergent='magma'):
```

Comment:
can you add a test where you set it to `mpl.cm.magma`. It should all just work but would be nice to explicitly test for that option.

Reply:
Just added an explicit test.